landingzones/views_api.py

```diff
@@ -227,6 +233,65 @@ def perform_create(self, serializer):
             raise APIException('{}{}'.format(ex_msg, ex))
 
 
+class ZoneUpdateAPIView(
```

Comment:
`ZoneUpdateAPIView` is missing from the API docs in `api_landingzones.rst`, please add.